coka/rate-limit-bundle/1.0/config/packages/oka_rate_limit.yaml

```diff
@@ -0,0 +1,3 @@
+oka_rate_limit:
+    configs:
+        - {path: ''}
```

Comment:
I like this config, because me as user are probably going to change that. 

:+1:

Reply:
Hi,
Cool, thanks